OpenTabletDriver.UX/Windows/DeviceStringReader.cs

```diff
@@ -165,17 +165,8 @@ private async Task<string> SendRequest(string strIndex, string strVid, string st
         {
             if (int.TryParse(strIndex, out var index) && index < 256 && index > 0)
             {
-                try
-                {
-                    if (int.TryParse(strVid, out var vid) && int.TryParse(strPid, out var pid))
-                        return await App.Driver.Instance.RequestDeviceString(vid, pid, index);
-                    else
-                        return await App.Driver.Instance.RequestDeviceString(index);
-                }
-                catch
-                {
-                    throw;
-                }
+                if (int.TryParse(strVid, out var vid) && int.TryParse(strPid, out var pid))
+                    return await App.Driver.Instance.RequestDeviceString(vid, pid, index);
```

Comment:
This breaks the send string to current tablet when we don't specify vid and pid, but specified index.

```suggestion
                if (int.TryParse(strVid, out var vid) && int.TryParse(strPid, out var pid))
                    return await App.Driver.Instance.RequestDeviceString(vid, pid, index);
                else
                    return await App.Driver.Instance.RequestDeviceString(index);
```

The suggestion might be broken on your implementation, but it still is a regression from master.

Reply:
I'm intentionally removing it in the first place, most of the use case is to get device strings from new devices rather than existing ones.